include/mbgl/map/map.hpp

```diff
@@ -37,6 +37,9 @@ class GlyphAtlas;
 class SpriteAtlas;
 class LineAtlas;
 class Environment;
+class AnnotationManager;
+
+typedef std::vector<LatLng> AnnotationSegment;
```

Comment:
The reasoning here is because the shapes API will have it; you pass vectors of `AnnotationSegment` objects to represent lines in a shape.